api-authorization/server/lib/authorization-middleware.js

```diff
@@ -22,7 +22,7 @@ export function authorizationMiddleware(req, res, next) {
     */
   const auth = req.get('Authorization');
   const token = auth.split('Bearer ')[1];
-  if (!token || !auth) {
+  if (!token || auth === undefined) {
```

Comment:
This doesn't fix the bug. `auth` can still be undefined on line 24 where it's referenced. Did you test this by sending an HTTP request without an Authorization header?